app/src/main/java/com/orange/ods/app/ui/modules/Module.kt

```diff
@@ -25,16 +24,25 @@ sealed class Module(
     @DrawableRes val imageRes: Int,
     @StringRes val descriptionRes: Int,
     val route: String,
-    val demoScreen: @Composable (upPress: () -> Unit) -> Unit,
-    val imageAlignment: Alignment = Alignment.Center,
+    val imageAlignment: Alignment = Alignment.TopCenter,
 ) {
+    companion object {
+        fun fromId(moduleId: Long?) = modules.firstOrNull { module -> module.id == moduleId }
```

Comment:
This method is unused at the moment. But we can keep it for future use if you want to.